### PR TITLE
Move to org wide hooks

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [x] Pushed the branch to main repo
* [x] CI passed on the branch

### CODE REVIEW

File: conda_forge_webservices/webapp.py [hunk 1]
```diff
@@ -207,7 +211,7 @@ def post(self):
             if 'head_commit' in body:
                 commit = body['head_commit']['id']
             # Only do anything if we are working with conda-forge, and a push to master.
-            if owner == 'conda-forge' and ref == "refs/heads/master":
+            if owner == 'conda-forge' and repo.endswith("-feedstock") and ref == "refs/heads/master":
```
Comment (jakirkham) [inline conda_forge_webservices/webapp.py]: Why limit it to `master`?

Comment (isuruf) [reply conda_forge_webservices/webapp.py]: This was there before. (This hook updates the team if master changes and we only look at master for updating the team)

**Comment by isuruf** [comment]
Well, I was careful. A thorough review is appreciated.

**Comment by jakirkham** [comment]
So I'm probably missing context here, but what will get by using the org webhook? Some events we don't get now? Easier to work with requests? Something else?

**Comment by beckermr** [comment]
Fewer api requests when making feedstocks.

**Comment by jakirkham** [comment]
Do we have a sense of how many requests we would save?

File: conda_forge_webservices/webapp.py (hunk 1 above)
Comment (beckermr) [inline conda_forge_webservices/webapp.py]: This should be `repo_name` as opposed to `repo`, right?

File: conda_forge_webservices/webapp.py [hunk 2]
```diff
@@ -230,7 +234,7 @@ def post(self):
             repo_name = body['repository']['name']
             owner = body['repository']['owner']['login']
             # Only do anything if we are working with conda-forge
-            if owner != 'conda-forge':
+            if owner != 'conda-forge' or not (repo == "staged-recipes" or repo.endswith("-feedstock")):
```
Comment (beckermr) [inline conda_forge_webservices/webapp.py]: ditto here I think

File: conda_forge_webservices/webapp.py [hunk 3]
```diff
@@ -257,7 +261,7 @@ def post(self):
             issue_num = body['issue']['number']
 
             # Only do anything if we are working with conda-forge
-            if owner != 'conda-forge':
+            if owner != 'conda-forge' or not (repo == "staged-recipes" or repo.endswith("-feedstock")):
```
Comment (beckermr) [inline conda_forge_webservices/webapp.py]: and here

**Comment by isuruf** [comment]
Added more tests so that bugs like before will be tested.

**Comment by beckermr** [comment]
Also it may be good to run a branch in staged recipes to slowly remove the old hooks.

**Comment by isuruf** [comment]
> We’ll need to add the new webhook a before merging

I already did

**Comment by isuruf** [comment]
Thanks @beckermr

**Comment by isuruf** [comment]
@jakirkham, forgot to answer your comment. staged-recipes is now failing at the stage where webhooks are added. For eg: https://travis-ci.com/conda-forge/staged-recipes/builds/147001015
This is one step towards fixing that.

**Pull request merged**